app/locale/en.coffee

```diff
@@ -1,7 +1,9 @@
 module.exports = nativeDescription: "English", englishDescription: "English", translation:
   new_home:
     title: "CodeCombat - Coding games to learn Python and JavaScript"
+    meta_keywords: "CodeCombat, python, javascript, Coding Games"
     meta_description: "Learn typed code through a programming game. Learn Python, JavaScript, and HTML as you solve puzzles and learn to make your own coding games and websites."
+    meta_og_url: "http://codecombat.com"
```

Comment:
While we're at it, this should be `https` instead of `http`

Reply:
oh yeah, updated